controllers/storagecluster/external_resources.go

```diff
@@ -353,21 +353,22 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 				scc = newCephBlockPoolStorageClassConfiguration(instance)
 			} else if d.Name == cephRgwStorageClassName {
 				rgwEndpoint := d.Data[externalCephRgwEndpointKey]
-				if err := checkEndpointReachable(rgwEndpoint, 5*time.Second); err != nil {
-					r.Log.Error(err, "RGW endpoint is not reachable.", "RGWEndpoint", rgwEndpoint)
-					return err
-				}
-				extCephObjectStores, err = r.newExternalCephObjectStoreInstances(instance, rgwEndpoint)
+				err := checkEndpointReachable(rgwEndpoint, 5*time.Second)
```

Comment:
I thought we discussed to store the rgw error, and then fail the reconcile later around line 363?
```suggestion
				rgwErr = checkEndpointReachable(rgwEndpoint, 5*time.Second)
```